src/Components/Nav/Nav.jsx

```diff
@@ -78,13 +85,34 @@ const Nav = () => {
         />
       </ul>
 
-      <div>
-        <NavButton
-          name={t("nav.settings")}
-          design={menuStyle}
-          icon={<SettingsIcon />}
-          link="/settings"
-        />
+      <div className="nav-legal">
+        <div
+          className={`nav-legal-wrapper ${!isLegalNoticeAvailable && "hidden"}`}
```

Comment:
why do you hide that via style and not via conditional jsx?

Reply:
I used this because isLegalNoticeAvailable is false by default which results in not rendering this component. This results in not calling the function, which checks whether the domain is available or not and can't trigger the submitCallback. So the component is not showing up although the domain is available. This can be bypassed by setting the variable to true by default, which I have done now. Now the link is rendered first and after that checked if domain is available. If it isn't available, the component will be stopped from rendering.